sites/libguides/libguides.html

```diff
@@ -53,3 +53,46 @@ <h4>Need more help? Ask Us!</h4>
     </ul>
   </div>  
 </section>
+
+<section id="homepage-sidebar">
+  <div id="col2" class="col-md-4 center">
+    <div id="homepage-sidebar-spring2020" class="full-width big-text">
```

Comment:
this has been added/switched to use [libguides](https://bu.libapps.com/libguides/groups.php?action=5&group_id=5217) [Boxes](https://bu.libapps.com/libguides/admin_c.php?g=361741&p=7433630), so it's no longer needed here.